cltk/phonology/utils.py

```diff
@@ -20,6 +20,8 @@ class Manner(AutoName):
     fricative = auto()
     trill = auto()
     spirant = auto()
+    affricate = auto()
```

Comment:
Good idea to improve this module. I wonder if this should be renamed to be more understandable.

Reply:
Yeah, operator overloading in C++ is really annoying because obscure.  The first time I saw 'cout << 'hello, world'` I felt manipulated :)  There's little defence, except to argue that one cat put together DSLs in languages that lack macro facilities.

Reply:
Well, I guess that one change is innocent enough. In terms of the name -- is `affricate` not understandable?  I don't know of another name for plosive-spirant sequences.